scripts/build.sh

```diff
@@ -658,8 +658,5 @@ elif [ "$COMPRESS_FORMAT" = "xz" ]; then
         echo "Out of memory? Trying again with single threads..."
         tar -cPJvf "$OUTPUT_PATH" "$WORK_DIR/wsa/$artifact_name" || abort
     fi
-elif [ "$COMPRESS_FORMAT" = "zip" ]; then
-    echo "Compressing with zip"
-    cp -r "$WORK_DIR/wsa/$artifact_name" "$OUTPUT_PATH" || abort
```

Comment:
@s1204IT Why you remove this line. We need it to copy file from the WORK_DIR to OUTPUT

Reply:
Sorry. I removed too much.
I'll fix it.